src/index.ts

```diff
@@ -29,6 +29,12 @@ const convertStateToEnv = (state: MediaState): Parameters<typeof matches>[1] =>
     return env;
 };
 
+function toCamelCase(str: string) {
+    return str.replace(/^-*/, "").replace(/-([a-z])/g, (match: string) => {
```

Comment:
FYI: `camelcase` is already in our dependencies. I think we could re-use it (same `"camelcase": "^5.3.1"` as our transitive dep `media-query-parser`)